magento/magento2
# Configurable product grid legend label overlapping to each other

### Description (*)

Configurable product grid legend label overlapping to each other

**Preconditions**

1. Magento 2.3.0
2. PHP 7.2

### Manual testing scenarios (*)

Step 1: Go to admin url
Step 2: login to admin panel
Step 3: click on Catalog link from left side then click on product link in sub menu 
 Step 4: Click on add product button dropdown and select simple product 
 Step 5: Fill simple product form and click create configuration button existing under Configurations  tab.
Step 6: Complete steps to create configuration (Popup will open)
 Step 6.1: Select Attributes then click next button 
 Step 6.2: Select Attribute Values then click in next button 
 Step 6.3: Bulk Images, Price and Quantity then click next button 
 Step 6.4: Summary screen will show and click on generate product button 
    Now you will see issue (ref actual result screenshot image)

### Actual result (*)

![issue](https://user-images.githubusercontent.com/26018716/50204075-8d726c80-0389-11e9-849d-832e91eb5e54.png)

### Expected result (*)

![sol](https://user-images.githubusercontent.com/26018716/50204090-9e22e280-0389-11e9-8eef-43cda3fd5b38.png)



### Contribution checklist (*)
